### PR TITLE
[FIX] documentation: allow newbies to find their newly-written modules

### PR DESCRIPTION
**Pull request by rim-odoo**
This is a recurring question for new developers at Odoo R&D ;)

### CODE REVIEW

File: doc/howtos/backend.rst [hunk 1]
```diff
@@ -115,6 +115,20 @@ or XML. The usage of most of those files will be explained along this tutorial.
...
+lead to unintuitive behavior: you write (the skeleton of) a new module,
+you add the *module directory* to the ``--addons-path`` and... you don't find
+it once the server is restarted.
+
+In order to update the list of available modules, switch to the *Developer
+mode*, then go to :menuselection:`Apps --> Update Apps List --> Update`.
+
+If you still don't find your module in the Apps list, this may be due to an
+incorrect module structure or a bad ``--addons-path``.
+
```
Comment (guewen) [inline doc/howtos/backend.rst]: tip: when you install or update a module through `-i` or `-u`, the list is automatically refreshed ;)

Comment (rim-odoo) [reply doc/howtos/backend.rst]: Even before 11.0 ?

For the records: at this step of the tutorial, `-i` and `-u` options are not explained yet

**Comment by mart-e** [comment]
Why making this PR for v9 anyway? As it is an improvement of the doc, so we can focus on v11 IMHO.

People starting to develop following this tutorial should already be a bit familiar with start and stopping a tutorial.
I think just adding a bullet point after scaffold

> *  execute `odoo-bin -i openacademy` to install your fresh module

If people followed the given command, their new module is already in the `addons/` folder anyway. We already introduce the `--addons-path` above.
I am not in favour of adding a new section that would make this very long tutorial even longer.

**Comment by rim-odoo** [comment]
@mart-e 

> Why making this PR for v9 anyway?

It is still a supported version and documentation will be forwardported ;)

> I think just adding a bullet point after scaffold
> 
>  > execute odoo-bin -i openacademy to install your fresh module
> 



> Maybe we can think about a troubleshooting page or something but I don't really think it is the place in this tutorial for that

It is, as **it is a recurring question from people following the tutorial**.

**Comment by rim-odoo** [comment]
up

**Comment by rim-odoo** [comment]
up @mart-e ;)

**Comment by mart-e** [comment]
You ups haven't made me change my mind about the v9 vs v11 :)
Syntactically the PR is correct but on the content itself, I would not bother to do it in v9 and just to the simplify version for v11.
But that is just my opinion...

**Pull request closed without merging**